packages/elements/src/components/ui/collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import { ReactNode } from "react";
+import { config, Spring } from "react-spring/renderprops-universal";
+import styles from "./Collapsible.module.css";
+import { Icon } from "../../..";
+import { faChevronDown } from "@fortawesome/free-solid-svg-icons";
+
+export interface CollapsibleProps {
+  label: string;
+  contentRight?: ReactNode;
+  collapsed?: boolean;
+  onCollapse?: () => void;
+  disabled?: boolean;
+}
+
+export const Collapsible: React.FC<CollapsibleProps> = ({
+  label,
+  contentRight,
+  collapsed = false,
+  onCollapse = function() {},
+  disabled = false,
+  children
+}) => {
+  return (
+    <div className={styles.collapsible} aria-expanded={!collapsed}>
+      <button
+        className={styles.header}
+        onClick={onCollapse}
+        disabled={disabled}
+      >
+        <div className={styles.headerText}>{label}</div>
+        <div className={styles.contentRight}>{contentRight}</div>
+        <Icon icon={faChevronDown} className={styles.chevron} size={8} />
```

Comment:
I think rotate and color is available as props. I'm not 100% sure though, but I think you can do this one without className.